conda_smithy/feedstock_tokens.py

```diff
@@ -343,14 +328,10 @@ def register_feedstock_token(user, project, token_repo, provider=None):
                 .replace("${GH_TOKEN}", github_token)
             )
             repo = git.Repo.clone_from(_token_repo, tmpdir, depth=1)
-            token_file = os.path.join(
-                tmpdir,
-                "tokens",
-                project + ".json",
-            )
+            token_file = str(Path(tmpdir, "tokens", f"{project}.json"))
 
             # append the token if needed
-            if os.path.exists(token_file):
+            if Path(token_file).exists():
                 with open(token_file, "r") as fp:
```

Comment:
I think we can use `token_file.open()` (instead of `open(token_file)`

Reply:
or even `json.loads(token_file.read_string())`

Reply:
I think you mean `.read_text()`, right?